mzLib/Proteomics/ProteolyticDigestion/ProteolyticPeptide.cs

```diff
@@ -97,12 +97,10 @@ internal IEnumerable<PeptideWithSetModifications> GetModifiedPeptides(IEnumerabl
             }
 
             // LOCALIZED MODS
-            var localizedModsOrderedByPositionThenByModification = Protein.OneBasedPossibleLocalizedModifications
+            foreach (var localizedMod in Protein.OneBasedPossibleLocalizedModifications
                 .SelectMany(kvp => kvp.Value.Select(value => (kvp.Key, value)))
                 .OrderBy(tuple => tuple.Key)
-                .ThenBy(tuple => tuple.value)
-                .ToList();
-            foreach (var localizedMod in localizedModsOrderedByPositionThenByModification)
```

Comment:
Sorry, I meant a custom comparer that considers the tuple's key and value simultaneously. So there would only be an OrderBy statement with a custom comparer, as opposed to an OrderBy().ThenBy().